.github/workflows/docs-ci.yml

```diff
@@ -64,6 +70,17 @@ jobs:
           remote_port: ${{ secrets.DOCS_PORT }}
           remote_user: ${{ secrets.DOCS_USER }}
           remote_key: ${{ secrets.DEEPHAVEN_CORE_SSH_KEY }}
+      - name: Deploy JavaScript/TypeScript docs
+        if: ${{ github.ref == 'refs/heads/main' }}
+        uses: burnett01/rsync-deployments@5.2
+        with:
+          switches: -avzr --delete
+          path: web/client-api/docs/build/documentation/
+          remote_path: deephaven-core/client-api/javascript/
```

Comment:
Not sure if remote path needs to exist before this job is run, just FYI. Obv sync expectations w/ deephaven.io.

Reply:
I'm not following - how could this be omitted, where would the CI job rsync the data to?

Reply:
Sorry - didn't mean to imply you don't need to set `remote_path`. I'm not sure if the _physical_ "remote path" needs to exist or not before this job will succeed. Just commenting on the fact that there is some coordination w/ deephaven.io that will be necessary.

Reply:
Ok, will verify with docs team.

Reply:
Running it creates the folder, nothing needs to be done. You should create a ticket to add a link in the sidebar back to this though once it merges though

Reply:
I will do that in a few minutes after seeing if I broke main...